Google.Api.Gax.Grpc/ClientBuilderBase.cs

```diff
@@ -368,6 +379,13 @@ protected virtual ChannelCredentials GetChannelCredentials()
                 GoogleCredential.GetApplicationDefault();
             GoogleCredential scoped = unscoped.CreateScoped(Scopes ?? GetDefaultScopes());
             GoogleCredential maybeWithProject = QuotaProject is null ? scoped : scoped.CreateWithQuotaProject(QuotaProject);
+
+            if (_useUseJwtAccessWithScopes && maybeWithProject.UnderlyingCredential is ServiceAccountCredential)
```

Comment:
And same comment as before.

Reply:
Perhaps I wasn't clear - I wasn't expecting this to be publicly mutable, although I guess we *could* do that. I was going to keep it read-only, but with subclasses able to override the default, effectively.

I think it's fine for this to be true, but ignored due to an explicit credential being set.